src/components/AuthorInfo.module.scss

```diff
@@ -0,0 +1,91 @@
+.main {
+  display: flex;
+  gap: 1em;
+
+  span {
+    line-height: 1;
+    white-space: nowrap;
+
+    h4 {
+      font-weight: 400;
+    }
+
+    p {
+      margin-bottom: 0.4em;
+    }
+
+    a {
+      position: relative;
+
+      &::after {
+        background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24' viewBox='0 0 24 24' fill='none' stroke='%23fff' stroke-width='2' stroke-linecap='round' stroke-linejoin='round'%3E%3Cpath d='M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6'%3E%3C/path%3E%3Cpolyline points='15 3 21 3 21 9'%3E%3C/polyline%3E%3Cline x1='10' y1='14' x2='21' y2='3'%3E%3C/line%3E%3C/svg%3E");
+        background-size: contain;
+        background-repeat: no-repeat;
+        width: 1.2em;
+        height: 1.2em;
+        content: '';
+        position: absolute;
+        margin-left: 0.3em;
+        bottom: 0.05em;
+      }
```

Comment:
Personally I am not a big fan of icons for external links. I feel that they clutter up the look and feel more than they help. I would still prefer a little Wikidata icon that would actually make clear what kind of link that is and save some space by making the text obsolete. Maybe we can try to use the Wikidata logo and blend it into the DraCor color the same way we do with the author images.

If, however, we decide to introduce generic external link icons we should be consistent and use them for all external links, not just in the author info.

Reply:
I hear you. It does look a bit cluttered. I'll think of a way to reintroduce the `data-link-label` with corresponding icons.